### PR TITLE
Update gradle versions.

### PR DESCRIPTION
**Pull request by IgorGanapolsky**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -1,3 +1,4 @@
 include ':hugo-annotations'
 include ':hugo-plugin'
 include ':hugo-runtime'
+include ':hugo-example'
```
Comment (JakeWharton) [inline settings.gradle]: This should not be here. The project cannot run in the same Gradle invocation as it depends on the plugin.

File: hugo-example/build.gradle [hunk 2]
```diff
@@ -7,8 +7,8 @@ buildscript {
       mavenLocal()
     }
 
-    classpath 'com.android.tools.build:gradle:1.3.0'
-    classpath 'com.jakewharton.hugo:hugo-plugin:1.2.2-SNAPSHOT'
```
Comment (JakeWharton) [inline hugo-example/build.gradle]: This was the correct value.

Comment (IgorGanapolsky) [inline hugo-example/build.gradle]: The app module was not building with 1.2.2-SNAPSHOT.  It only built when I
changed it to the value pointed out in the README.

Comment (JakeWharton) [inline hugo-example/build.gradle]: You need to install it into your local Maven repository with `./gradlew install` first. Gradle projects cannot use plugins which are build in the same project.

**Comment by IgorGanapolsky** [comment]
Squashed commit.

**Comment by JakeWharton** [comment]
I think you need to update `.travis.yml` to point at the new version as well.

**Comment by IgorGanapolsky** [comment]
The problem is Travis will fail with 
> failed to find Build Tools revision 23.0.1

**Comment by JakeWharton** [comment]
That means the platform-tools needs updated as well. Or was it just tools. I can't remember.

**Comment by IgorGanapolsky** [comment]
Updated.

File: .travis.yml [hunk 3]
```diff
@@ -6,8 +6,11 @@ jdk:
 
 android:
   components:
-    - build-tools-21.1.2
-    - android-21
+    - build-tools-23.0.1
+    - android-23
+
+    # Update Android SDK Tools
+    - tools
```
Comment (JakeWharton) [inline .travis.yml]: I'm surprised this doesn't have to go first, as I thought these were executed in order.

Comment (JakeWharton) [inline .travis.yml]: Maybe Travis special cases it?

**Pull request merged**